views/docs/section.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * @psalm-scope-this XdebugDotOrg\Model\DocsSection
+ */
+XdebugDotOrg\Controller\TemplateController::setTitle('Xdebug: Documentation » ' . $this->title);
+?>
+
+<h1>Documentation</h1>
+
+<?= XdebugDotOrg\Controller\TemplateController::default_menu()->render() ?>
+
+» Documentation for: <b>Xdebug 2</b><br />
+» Feature: <b><?= $this->title ?></b><br />
+<hr />
+
+<p class='intro'><?= $this->description ?></p>
+<hr class='light'/>
+<?= $this->text ?>
+<hr style='clear: both'>
+
+<?php if ($this->related_settings) : ?>
+	<h2>Related Settings</h2>
+	<div class='settings'>
+		<?php foreach ($this->related_settings as $setting) : ?>
+			<?= XdebugDotOrg\Controller\Docs\SettingsController::single($setting)->render() ?>
+		<?php endforeach ?>
```

Comment:
Can't say I'm a big fan of these "old style" looping constructs and short-echo-tags ;-)

Reply:
so this is a house style for views - in this particular context (with no HTML) it's slightly more confusing, but the idea is to be able to easily distinguish template code from other code.

Reply:
Yeah, I don't mind too much.